pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -1157,12 +1155,12 @@ func (qjm *XController) ScheduleNext(qj *arbv1.AppWrapper) {
 				// Assume preemption will remove low priroity AWs in the system, optimistically dispatch such AWs
 
 				if aggqj.LessEqual(resources) {
-					//TODO: should we turn-off histograms?
-					unallocatedHistogramMap := qjm.cache.GetUnallocatedHistograms()
-					if !qjm.nodeChecks(unallocatedHistogramMap, qj) {
-						klog.Infof("[ScheduleNext] [Agent Mode] Optimistic dispatch for AW '%s/%s' requesting aggregated resources %v histogram for point in-time fragmented resources are available in the cluster %s",
-							qj.Name, qj.Namespace, qjm.GetAggregatedResources(qj), proto.MarshalTextString(unallocatedHistogramMap["gpu"]))
-					}
+					//cache is turned-off, refer issue: https://github.com/project-codeflare/multi-cluster-app-dispatcher/issues/588
```

Comment:
The histograms are not part of the cache issue, right?

Reply:
Yes, they call a method inside the cache package which constructs histograms

Reply:
Ok, I believe we should move away from histograms anyway as I don't see why we can't use the ACTUAL topology of the cluster to make decisions.